### ISSUE TITLE
Re-work the web app to toggle between different edge lists

### ISSUE THREAD

**Issue by shreyapandit** [ASL-LEX/asl-lex#11]
- [x] Remove the Heroku call and have jsons locally
- [ ] Provide an option on the UI to reload another edge list. Should be at the bottom of the About tab.
- [ ] Re-render the graph based on the new edge list.

Pick up after #8 and #9  is done.

**Comment by shreyapandit** [comment]
Add a tab similar to filter and about, called Customize for now. We can discuss better UI options in our next meeting

**Comment by shreyapandit** [comment]
We are putting this on hold for now because every other edge list except the default has millions of edges which we can't display in a scalable manner with the current graph. Since each of the criteria have missingFeatures=0, the subgraphs are completely connected.
Hence, we might have to make each group a hub (a big node) and when the user clicks to expand that, we can look at edge bundling for displaying edges _within_ that subgraph.

An example of edge bundling can be found here: https://bl.ocks.org/mbostock/1044242

<img width="1184" alt="Screen Shot 2019-09-04 at 11 33 16 AM" src="https://user-images.githubusercontent.com/836757/64269401-d19da100-cf07-11e9-8e2f-89f5f6a4cea7.png">

**Comment by shreyapandit** [comment]
Closing this as invalid, we will be looking at multi-level display of node hubs as inspired by https://observablehq.com/@d3/zoomable-circle-packing
